src/panels/config/zwave/ha-config-zwave.js

```diff
@@ -563,6 +557,11 @@ class HaConfigZwave extends LocalizeMixin(PolymerElement) {
     };
   }
 
+  nodeMoreInfo(ev) {
+    ev.preventDefault();
```

Comment:
I don't think that this is necessary?

Reply:
I originally copied from the more-info code in dev-states, but tested it without this line and it looks like it still works, so I'll remove it.